spec/system/kudos_spec.rb

```diff
@@ -152,6 +152,19 @@
       expect(Kudo.count).to eq 0
     end
 
+    it 'not being able to remove a kudo 5 minutes after it was sent' do
+      expect(Kudo.count).to eq 1
+      visit root_path
+
+      travel 6.minutes
+
+      visit root_path
+      click_link 'Destroy'
+
+      expect(page).not_to have_content 'Kudo was successfully destroyed.'
```

Comment:
let's check here for an error message

Reply:
Actually clicking this link 5 minutes after kudo creation will not provide to appearing any error. I have understood the AC: "If someone opens a page with a kudo more than 5 minutes after it was sent, the edit and delete links are disabled" that these links should have been seen but at the same time disabled.My solution guarantees that 5 minutes after sent a kudo, you can click on destroy link but its will do nothing -> this link is disabled ('grayed out'). Thus I can only check if clicking destroy link provides to any action and therefore cannot test if any error appears :(

Reply:
This is another case, where i'm not reading correctly AC 👀 
But in this case, maybe we can check if the page has a disabled button/link 'Destroy'

`expect(page).to have_button('Destroy', disabled: true)`

Reply:
It's great thought in my opinion. I had tried do that before I created PR but I failed then. Only now with your help I came up with an idea that of course I have to change destroy link into a button (like I did before) but I should also change the syntax in index.html.erb. Working syntax for making links disabled is `class:'btn btn-outline-primary disabled'`. On the first try I used for buttons the same syntax so the tests did not see when buttons was made disabled. Now I changed the syntax for buttons into: `class:'btn btn-outline-danger', disabled: true,` and it works!!!!! Thanks a lot :)